scipy/io/wavfile.py

```diff
@@ -278,10 +301,16 @@ def read(filename, mmap=False):
         else:
             fid.seek(0)
 
-    return fs, data
+    result = [fs, data]
+    if return_cues:
+        result.append(cue)
+        result.append(cuelabels)
+    if return_pitch:
+        result.append(pitch)
+    return tuple(result)
```

Comment:
Added an example of use of the proposed API [here](https://github.com/scipy/scipy/pull/6852#issuecomment-266532025).